docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -12,9 +12,9 @@ weight = -16
 
 # What's new in Grafana v7.2
 
-This topic includes the release notes for the Grafana v7.2, which is currently in beta. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+This topic includes the release notes for the Grafana v7.2. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
 
-This article is not complete yet. We will be adding new details over the coming days as we get closer to a stable release.
+This article is not complete yet. We will be adding new details over the coming days.
```

Comment:
```suggestion
```

It should be complete by now, seems like updating the header was just forgotten about.